blog/en/blog/2023/12/26/access-kafka-by-apisix.md

```diff
@@ -0,0 +1,271 @@
+---
+title: Access the Kafka Cluster by APISIX Gateway
+authors:
+  - name: Meng Yan
+    title: Author
+    url: https://github.com/yanmxa
+keywords:
+  - Message Queue
+  - Kafka
+  - Gateway
+  - Strimzi
+description: A few days ago, I tried to add a proxy to the kafka cluster, so I can leverage the proxy to manage the kafka authentication and authorization. Next, I will show how to use the APISIX to develop a customize authorization plugin for the kafka cluster.
+tags: [Ecosystem]
+---
```

Comment:
@Yilialinn @JYan00 will you add a cover image now or later?

Reply:
https://static.apiseven.com/uploads/2023/12/29/oJyQz1c4_A&K_Cover.png